modules/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/step11_rgmb_homo.md

```diff
@@ -0,0 +1,88 @@
+# Reactor Geometry Mesh Builder Example: Homogeneous Sodium-Cooled Fast Reactor Core (ABTR)
+
+This example illustrates the use of RGMB mesh generators to define a 3D hexagonal geometry core with homogeneous assemblies (ABTR ([!cite](shemon2015abtr)), constructed earlier in this tutorial using base mesh generators).
+
+!media tutorial04_meshing/rgmb_abtr_stepbystep.png
```

Comment:
@shikhar413  - may need to update the 3D image in this figure to reflect the axial material of the shield/reflector (whatever we saw the other day that may not be consistent). looks like the bottom layer is all the same then the top layer isn't.

Reply:
Will send Aaron an updated image